bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/SpectralDistribution.scala

```diff
@@ -35,88 +35,194 @@ sealed trait Library extends SpectralDistribution {
+  case object M3III_new extends LibraryStar("M3III (115nm-4.91um)", "M3III_pickles_irtf")
+  case object M3V       extends LibraryStar("M3V (300nm-6.0um)",    "M3V")
+  case object M3V_new   extends LibraryStar("M3V (115nm-5.07um)",   "M3V_pickles_irtf")
+  case object M4III     extends LibraryStar("M4III (115nm-5.02um)", "M4III_pickles_irtf")
+  case object M4V       extends LibraryStar("M4V (115nm-5.08um)",   "M4V_pickles_irtf")
+  case object M5V       extends LibraryStar("M5V (115nm-4.18um)",   "M5V_pickles_irtf")
+  case object M6III     extends LibraryStar("M6III (300nm-6.0um)",  "M6III")
+  case object M6III_new extends LibraryStar("M6III (115nm-5.02um)", "M6III_pickles_irtf")
+  case object M6V       extends LibraryStar("M6V (300nm-6.0um)",    "M6V")
+  case object M7III     extends LibraryStar("M7III (115nm-5.02um)", "M7III_pickles_irtf")
+  case object M8III     extends LibraryStar("M8III (115nm-5.02um)", "M8III_pickles_irtf")
+  case object M9III     extends LibraryStar("M9III (300nm-6.0um)",  "M9III")
+  // sub-dwarfs
+  case object sdB       extends LibraryStar("sdB (100nm-32um model)",  "sdB_calspec")
+  case object sdF8      extends LibraryStar("sdF8 (171nm-2.48um)",     "sdF8_calspec")
+  case object sdO       extends LibraryStar("sdO (115nm-1.02um)",      "sdO_calspec")
+  // white dwarfs
+  case object DA08      extends LibraryStar("DA.8 (100nm-32um model)",  "DA08_calspec")
+  case object DA09      extends LibraryStar("DA.9 (100nm-30um model)",  "DA09_calspec")
+  case object DA12      extends LibraryStar("DA1.2 (100nm-30um model)", "DA12_calspec")
+  case object DA15      extends LibraryStar("DA1.5 (100nm-30um model)", "DA15_calspec")
+  case object DA18      extends LibraryStar("DA1.8 (100nm-27um model)", "DA18_calspec")
+  case object DA24      extends LibraryStar("DA2.4 (100nm-32um model)", "DA24_calspec")
+  case object DA28      extends LibraryStar("DA2.8 (114nm-1.71um)",     "DA28_calspec")
+  case object DA30      extends LibraryStar("DA3.0 (114nm-1.71um)",     "DA30_calspec")
+  case object DA31      extends LibraryStar("DA3.1 (114nm-1.71um)",     "DA31_calspec")
+  case object DA33      extends LibraryStar("DA3.3 (114nm-1.71um)",     "DA33_calspec")
+  case object DA36      extends LibraryStar("DA3.6 (114nm-1.71um)",     "DA36_calspec")
+  case object DA38      extends LibraryStar("DA3.8 (114nm-1.71um)",     "DA38_calspec")
+  case object DA48      extends LibraryStar("DA4.8 (114nm-1.71um)",     "DA48_calspec")
+  case object DA57      extends LibraryStar("DA5.7 (114nm-1.71um)",     "DA57_calspec")
+  case object DBQ40     extends LibraryStar("DBQ4 (100nm-32um model)",  "DBQ40_calspec")
+  case object DBQA50    extends LibraryStar("DBQA5 (100nm-32um model)", "DBQA50_calspec")
+  case object DO20      extends LibraryStar("DO2 (114nm-1.02um)",       "DO20_calspec")
+  // cool star models
+  case object T2800K    extends LibraryStar("T=2800K (300nm-6um model)",  "t2800k")
```

Comment:
All cool star models changed casing from "TXXXXK" "tXXXXk"

Reply:
Thanks, I missed that.

Reply:
Ah, these were fixed in a later commit (d85de18, "fix a few missed entries in SpectralDistribution.scala").

Reply:
Odd I reviewed in github but the PR is so large it could have some problems. I'll check again locally

Reply:
you're right this is correct on the version I get from your branch